alembic/latiss/versions/bf7ba4e029d4_add_exposure_quicklook_to_latiss_and_.py

```diff
@@ -0,0 +1,617 @@
+"""add exposure quicklook to latiss and updates including pixelscale psf mount jitter and magnitude limits
+
+Revision ID: bf7ba4e029d4
+Revises: e9fd2cdb9c12
+Create Date: 2024-11-12 18:33:35.448770+00:00
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql, postgresql
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "bf7ba4e029d4"
+down_revision: Union[str, None] = "e9fd2cdb9c12"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
```

Comment:
@JeremyMcCormick 
I've committed this to show you what we delete, rather than committing the cleaned up version. 
we remove: 
this `op.create_table("ccdvisit1"...` to line 56, 
and the `op.create_table("visit1", ...` to line 404